internal/server/server_edgex.go

```diff
@@ -0,0 +1,33 @@
+//go:build edgex
+
+// Copyright 2022-2024 EMQ Technologies Co., Ltd.
```

Comment:
let me know what i should do with these copyrights, i must have copied/pasted this. should I put my company in here? EdgeX Foundry? OpenZIti? etc. thx

Reply:
If it is originally composed by you in your working time, just put your company here.

Reply:
thank you. updated.